pkg/virt-handler/vm_test.go

```diff
@@ -837,6 +837,54 @@ var _ = Describe("VirtualMachineInstance", func() {
 			))
 		})
 
+		It("should add access credential synced condition when deprecated method is used for credentials update", func() {
```

Comment:
This should probably state that the event has to be of type `Warning`?

Reply:
Maybe this and the test case before can be de-duped with a Table.

Reply:
done